syntax.in

```diff
@@ -1,31 +0,0 @@
-<cmpstmt> ::= <stmt> | <stmt> ; <cmpstmt>
-<stmt> ::= <assignStmt> | <structStmt> | <funcDeclStmt>
-<assignStmt> ::= <identifier>=<expression>
```

Comment:
Operators, separators and reserved words should be written between quotes to be better seen

Reply:
fixed